modules/templates/helper.go

```diff
@@ -620,6 +622,16 @@ func Str2html(raw string) template.HTML {
 	return template.HTML(markup.Sanitize(raw))
 }
 
+// Md2html render Markdown text to HTML (non-sanitize)
+func Md2html(raw string) template.HTML {
+	var err error
+	var RenderedContent string
+	if RenderedContent, err = markdown.RenderString(&markup.RenderContext{}, raw); err != nil {
```

Comment:
I can see the call to `markup.RenderCommitMessage(&markup.RenderContext{` in this file.

Do we need `markup` or `markdown` actually?

Reply:
this is something i need to actually check, I used implementation from issue comments as a base of my PR and it had `markdown.RenderString(&markup.RenderContext{` but it might actually work properly with `markup` instead of `markdown`

Reply:
Seems `markup.RenderString` doesn't render markdown, so it actually has to be `markdown.RenderString`
![qq3](https://user-images.githubusercontent.com/19575594/146157735-bb8c372c-baa4-4c16-8218-8de15be6d62d.png)